lib/harmonium/table.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Harmonium.Table do
+  @moduledoc false
+
+  import Phoenix.HTML.Tag
+
+  def table(opts \\ [], do: block) do
+    table_element(:table, [class: "rev-Table"], opts, do: block)
```

Comment:
If we change all of these to...
```
def table_data(opts \\ []) do
    {block, opts} = Keyword.pop(opts, :do, nil)
    table_element(:td, [class: "rev-Table"], opts, do: block)
end
```
Then we can do something like...
```
<%= table_data colspan: 5 %>
<%= table_data do: sum %>
```
otherwise you have to do 
```
<%= table_data colspan: 5 do %>
  <%= sum %>
<% end %>
```